app/frontend/javascript/file-list/components/FileList.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="list-group list-group-flush" @click="handleClick">
```

Comment:
Did you find out why this is not being executed when the list is clicked?

Reply:
I believe the way it was handling the click before was by using another element's click event, which iwraps this element. The previous code is as follows which was in the index file of this component
`  if (document.getElementById('files-list')) {
    /* The files-list element isn't on all pages. So only initialize our
     * Vue app if we actually find it */
    let app = new Vue({
      el: '#files-list',
      render: (h) => h(FileList),
    })
    document.getElementById('files-tab-link').addEventListener('click', function () {
      app.$children[0].fetchData()
    })
  }
})`
I think it’s better to avoid that dependency here. Also, in the scenario where we are in the 'Files' tab and upload a new file, the page currently reloads, which causes the focused tab to switch to another one—likely the default tab. Therefore mount will be called anyway which avoids the need to do handle click. Hope it makes sense

Reply:
But, I think we are reloading the the page, the components will be re-mounted any way, right?

Reply:
Te component is loaded this way and the click was originally linked to the 'link' element with id 'files-tab-link'
 <li class="nav-item">
      <a id='files-tab-link' class="nav-link" href="#files_tab" data-toggle="tab" role="tab" data-plate-view="files-view">Files</a>
    </li>

Reply:
Yes, console.log confirms it

Reply:
@BenTopping, @StephenHulme - It would be great if one of you could also test whether the file upload is correctly displayed once it is in UAT, please.